stratum/hal/lib/common/phal_interface.h

```diff
@@ -124,6 +124,18 @@ class PhalInterface {
   virtual ::util::Status GetFrontPanelPortInfo(
       int slot, int port, FrontPanelPortInfo* fp_port_info) = 0;
 
+  // Gets the information about the optical transceiver module by querying TAI
+  // for the given node_id and port_id. This method is expected to return
+  // error if there is no related optics module inserted yet.
+  virtual ::util::Status GetOpticalTransceiverInfo(
+      uint64 node_id, uint32 port_id, OpticalChannelInfo* oc_info) = 0;
```

Comment:
I think we have to clarify on the notion if node and port IDs for optical ports. As far as I understand, nodes are close to modules and ports would be individual transceivers?